action.yml

```diff
@@ -25,6 +33,14 @@ inputs:
     description: "Install kubectl"
     default: "yes"
     required: false
+  install-vals:
+    description: "install vals"
+    default: "yes"
```

Comment:
For those who don't need this tool, I want to set the default to false to shorten the installation time.